.gitignore

```diff
@@ -1,4 +1,4 @@
-/target
+target/
```

Comment:
(suggestion, I am ok with this being merged):
If we have a java package or resource directory called "target", this ignore pattern "target" (or "target/") will exclude that.
The "/target" ignore pattern doesn't have that problem, but it needs a .gitignore file in every maven module.

The original consistent approach was to always favor "/target" and more .gitignore files over this approach, but that may have changed in some subprojects.